notifications/listener_test.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 func TestListener(t *testing.T) {
-	registry := storage.NewRegistryWithDriver(inmemory.New())
+	registry := storage.NewRegistryWithDriver(inmemory.New(), distribution.GlobalScope)
```

Comment:
I was thinking of a use case where a registry will limit which repositories it is serving. This may certainly be a non-feature.